src/huggingface_hub/serialization/_tensorflow.py

```diff
@@ -63,11 +63,11 @@ def split_tf_state_dict_into_shards(
         state_dict,
         max_shard_size=max_shard_size,
         filename_pattern=filename_pattern,
-        get_tensor_size=get_tensor_size,
+        get_storage_size=get_tf_storage_size,
     )
 
 
-def get_tensor_size(tensor: "tf.Tensor") -> int:
+def get_tf_storage_size(tensor: "tf.Tensor") -> int:
```

Comment:
Do we have all the equivalent torch methods for TensorFlow? Or is that not necessary?

Reply:
Not yet no. Let's build for torch first and then expand to TF after if needed